### DISCUSSION TITLE
Whenever I push in RStudio, Github says it's coming from a username other than my own.

### DISCUSSION THREAD

**Discussion by MerZielonka** [community/community#32993] [Other Feature Feedback, Questions, & Ideas]
I've done the who am I command in RStudio and it always gives me my correct username, but GitHub says another username is publishing my code in my repository.

**Comment by airtower-luna** [comment]
> GitHub says another username is publishing my code in my repository.

What do you mean by that? If the commits look like another user's, the author email address on the machine is probably incorrect, see [Set commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) for how to fix it.
